src/app/components/TranscriptExperiment/SignPost/svgs.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const FanSvg = ({ className }: { className?: string }) => (
```

Comment:
Let's remove Svg from the name
```suggestion
const Fan = ({ className }: { className?: string }) => (
```